plugins/bc-mail/tests/BcMailPluginTest.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+namespace BcMail\Test;
+
+use BaserCore\TestSuite\BcTestCase;
+use BcMail\BcMailPlugin;
+use BcMail\Service\Admin\MailContentsAdminServiceInterface;
+use BcMail\Service\Admin\MailFieldsAdminServiceInterface;
+use BcMail\Service\Admin\MailMessagesAdminServiceInterface;
+use BcMail\Service\Front\MailFrontServiceInterface;
+use BcMail\Service\MailConfigsServiceInterface;
+use BcMail\Service\MailContentsServiceInterface;
+use BcMail\Service\MailFieldsServiceInterface;
+use BcMail\Service\MailMessagesServiceInterface;
+use Cake\Core\Container;
+
+class BcMailPluginTest extends BcTestCase
+{
+    public function setUp(): void
+    {
+        parent::setUp();
+        $this->Plugin = new BcMailPlugin(['name' => 'BcMail']);
```

Comment:
@thangnnmd đặt lại tên biến nhé a.$this->BcMailPlugin

Reply:
@HungDV2022 anh sửa rồi nhé